doc/sphinx-guides/source/admin/troubleshooting.rst

```diff
@@ -32,49 +32,49 @@ A User Needs Their Account to Be Converted From Institutional (Shibboleth), ORCI
 
 See :ref:`converting-shibboleth-users-to-local` and :ref:`converting-oauth-users-to-local`.
 
-.. _troubleshooting-glassfish:
+.. _troubleshooting-payara:
```

Comment:
I'm ok leaving this section as troubleshooting Payara, so all the references inside are fine). (some of this troubleshooting may actually be more generic, but it's all under this header)

Reply:
Cool, me too.